css-layout-classes/styles.css

```diff
@@ -0,0 +1,40 @@
+body {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.row {
+  display: flex;
+}
+
+.four {
+  display: flex;
+}
+
+.pics {
+  max-width: 25%;
+}
+
+.one-third {
+  width: calc(100%/3);
+  margin: 1rem;
```

Comment:
layout classes should only have css applied that's relative to the layout. So `margin` should be applied to a separate class, that way you can reuse this class multiple times without having to worry about spacing get screwed up.